.travis.yml

```diff
@@ -30,11 +28,8 @@ script:
   - qiita_env make --no-load-ontologies
   - if [ ${TEST_ADD_STUDIES} == "True" ]; then test_data_studies/commands.sh ; fi
   - if [ ${TEST_ADD_STUDIES} == "False" ]; then qiita_test_install ; fi
-  - if [ ${TEST_ADD_STUDIES} == "False" ]; then nosetests --with-doctest --with-coverage ; fi
+  - if [ ${TEST_ADD_STUDIES} == "False" ]; then nosetests -v --with-doctest --with-coverage ; fi
```

Comment:
No idea, every single time when I removed that flag the suite would fail in a very *mysterious* way, see:

![screen shot 2015-03-20 at 7 06 29 pm](https://cloud.githubusercontent.com/assets/375307/6768087/8cdd06f6-d014-11e4-83a9-730397352ba4.png)

----

Or see the build of the second to last commit.